packages/extension-ui/src/components/AccountNamePasswordCreation.tsx

```diff
@@ -180,44 +168,50 @@ function AccountNamePasswordCreation({
 }
 
 const Container = styled.div`
-  & > :not(:last-child):not(:first-child) {
-    margin-bottom: 24px;
-  }
+  margin-right: 8px;
+  margin-bottom: auto;
 `;
 
-export default React.memo(styled(AccountNamePasswordCreation)`
-  margin-right: 8px;
+const StyledHeader = styled(Header)`
+  margin-bottom: 32px;
+`;
+
+const StyledAddress = styled(Address)`
+  margin-bottom: 24px;
+`;
 
-  .spacer {
-    height: 16px;
+const InputsWrapper = styled.div`
+  & > * {
```

Comment:
Isn't
```suggestion
  & > *:not(:last-child) {
```
needed?

Reply:
I think I want margin on the last child as well, to space properly from footer